crates/api_crud/src/post/update.rs

```diff
@@ -52,8 +52,8 @@ impl PerformCrud for EditPost {
 
     // Fetch Iframely and Pictrs cached image
     let data_url = data.url.as_ref();
-    let (iframely_title, iframely_description, iframely_html, pictrs_thumbnail) =
-      fetch_iframely_and_pictrs_data(context.client(), data_url).await;
+    let (iframely_response, pictrs_thumbnail) =
+      fetch_iframely_and_pictrs_data(context.client(), data_url).await?;
```

Comment:
I guess these need to be results (or options if they aren't already), since they could come back with errors.